tests/metal/texture-binding-index.slang

```diff
@@ -0,0 +1,24 @@
+//TEST:REFLECTION(filecheck=REFLECT): -stage compute -entry computeMain  -target metal
```

Comment:
Would be good to include a SIMPLE test that checks for the actual binding attributes in the generated metal source.

Reply:
Of course, I will made the change